server/rest/annotation.py

```diff
@@ -160,6 +161,33 @@ def createAnnotation(self, item, params):
                     exc.args, ))
 
     @describeRoute(
+        Description('Copy an annotation from an item to an other.')
+        .param('annotationId', 'The ID of the annotation.', paramType='path',
+               required=True)
+        .param('itemDest', 'Pass this to move the annotation to this item.',
+               required=True)
+        .errorResponse('Write access was denied for the item.', 403)
+        .errorResponse('Validation Error: JSON doesn\'t follow schema.')
+    )
+    @access.user
+    @loadmodel(model='annotation', plugin='large_image')
+    @filtermodel(model='annotation', plugin='large_image')
+    def copyAnnotation(self, annotationId, itemDest, params):
+        user = self.getCurrentUser()
+        annotation = self.model('annotation', 'large_image').load(annotationId, region=params)
+        if annotation is not None:
+            itemSrc = self.model('item').load(annotation.get('itemId'), force=True)
+        if itemSrc is not None:
+            self.model('item').requireAccess(
+                item, user=user, level=AccessType.READ)
+        if itemDest is not None:
+            self.model('item').requireAccess(
+                item, user=user, level=AccessType.WRITE)
+
+        return self.model('annotation', 'large_image').createAnnotation(
+            itemDest, user, annotation)
```

Comment:
This block won't work as is.  `itemDest` (or `itemId` when you rename it) is an ID string not a document.  What you want to do is more like this:
```python
itemDest = self.model('item').load(
    itemId, user=user, level=AccessType.WRITE, exc=True)

return self.model('annotation', 'large_image').createAnnotation(
    itemDest, user, annotation)
```

Also, you don't need to check if it is `None`.  The decorators above will assert that it exists.